app/src/main/java/com/elavon/converge/config/Config.java

```diff
@@ -0,0 +1,58 @@
+package com.elavon.converge.config;
+
+public class Config {
```

Comment:
config class coming from `app/src/main/res/raw/config_live.json` and `app/src/main/res/raw/config_test.json`